dist/redhat/build_rpm.sh

```diff
@@ -65,10 +65,14 @@ pkg_install git
 pkg_install python3
 pkg_install python3-devel
 pkg_install python3-pip
+pkg_install gcc
 
 echo "Running unit tests"
 cd tests
 sudo pip3 install pyyaml==5.3
+sudo pip3 install psutil
+sudo pip3 install distro
+sudo pip3 install traceback-with-variables
 python3 -m unittest test_scylla_configure.py
```

Comment:
only the build_rpm.sh has the unittest running ? the build_deb.sh doesn't ?

Reply:
the idea was to force scylla-pkg run them at build time. (before we using github actions, and before SMI has it's own next process)

also seem like it's not running the whole suite of tests... only test_scylla_configure.py.

maybe this section can be removed.

Reply:
Dropped unit section from build_rpm.sh